content/docs/cli-overview.md

```diff
@@ -84,7 +85,7 @@ By default the mutation will not change the content of the files.
 
 Takes the following options,
 
-| Argument  | Description                                                                                                                                                                                                                                                        |
-| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| `--clean` | When this flag is used, it really submits the Graphql mutations to the file system. This means that it will clean out any fields that are not defined in your schema. It is a good practice to do a `git commit` before doing this so one can undo changes easily. |
-|           |
+| Argument         | Description                                                                                                                                                                                                                                                        |
+| ---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `--clean`        | When this flag is used, it really submits the Graphql mutations to the file system. This means that it will clean out any fields that are not defined in your schema. It is a good practice to do a `git commit` before doing this so one can undo changes easily. |
```

Comment:
```suggestion
| `--clean`        | When this flag is used, it submits actual Graphql mutations to the file system. This means that it will clean out any fields that are not defined in your schema. It is a good practice to do a `git commit` before doing this so unintended changes can be easily undone. |
```